template/src/components/kit/swiper/Swiper.tsx

```diff
@@ -0,0 +1,92 @@
+import { ContentStyle, FlashList, FlashListProps } from '@shopify/flash-list';
+import { View } from 'react-native';
+import Animated from 'react-native-reanimated';
+import React, { Ref, useImperativeHandle, useMemo, useRef } from 'react';
+import { useCenteredTileOffsets } from './hooks';
+
+export interface SwiperProps<TItem>
+  extends Omit<
+    FlashListProps<TItem>,
+    | 'contentContainerStyle'
+    | 'snapToOffsets'
+    | 'horizontal'
+    | 'estimatedItemSize'
+  > {
+  containerWidth: number;
+  containerHorizontalPadding: number;
+  tileWidth: number;
+  gap: number;
+  contentContainerStyle?: Pick<
+    ContentStyle,
+    'backgroundColor' | 'paddingVertical' | 'paddingTop' | 'paddingBottom'
+  >;
+  footerWidth?: number;
+  forwardedRef?: Ref<SwiperMethods>;
+}
+
+export type SwiperMethods = { scrollToTop: (animated?: boolean) => void };
```

Comment:
Maybe I'm missing something but shouldn't it be `scrollToLeft` or `scrollToStart` since the list is horizontal?

Reply:
You're right, scrollToStart is much better hahaha